test/functional/testlib.bash

```diff
@@ -802,7 +802,31 @@ create_bundle() {
 
 		EOM
 	}
-	
+
+	add_dirs_recursively() {
```

Comment:
not a big deal, but this doesn't actually do any recursion, it's all iterative.

Reply:
that's true, this function doesn't call itself, I'll rename it